sarpy/io/complex/sicd_elements/PFA.py

```diff
@@ -192,12 +192,12 @@ def project_to_image_plane(points):
 
         # compute polar angle of sensor position in image plane
         ip_range = image_plane_positions - SCP
-        ip_range = ip_range/numpy.linalg.norm(ip_range)[:, numpy.newaxis]
+        ip_range = (ip_range/numpy.linalg.norm(ip_range))[:, numpy.newaxis]
         k_a = -numpy.arctan2(ip_range.dot(ip_y), ip_range.dot(ip_x))
 
         # compute the spatial frequency scale factor
         range_vectors = positions - SCP
-        range_vectors = range_vectors/numpy.linalg.norm(range_vectors)[:, numpy.newaxis]
+        range_vectors = (range_vectors/numpy.linalg.norm(range_vectors))[:, numpy.newaxis]
```

Comment:
This is not quite right. I'll fix this in a separate branch, and merge things in. I'm going to reject this merge request, just because it's easier to fix it that way.